### PR TITLE
LPS-106852 portal-search-web: Add ADTs for tag facet widget

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-106852

@jonmak08

### CODE REVIEW

**Comment by oliv-yu** [comment]
ci:test:search:norebase

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -161,6 +163,7 @@ set-up-its-destination-to-make-it-visible=Set up its destination to make it visi
 site-parameter-name=Site Parameter Name
 sort-by=Sort by
 specific-facet-configuration=Specific Facet Configuration
+tag-facet-portlet-instance-configuration-name=Tag Facet Configuration
```
Comment (jonmak08) [inline modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: Shouldn't the language key and the value match?

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -10,6 +10,8 @@ any-category=Any Category
 any-folder=Any Folder
 any-site=Any Site
 any-user=Any User
+asset-tag-search-facet-display-context=Tag Facet Display Context
```
Comment (jonmak08) [inline modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: Shouldn't the language key and the value match?

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties [hunk 3]
```diff
@@ -10,6 +10,8 @@ any-category=Any Category
...
+asset-tag-search-facet-display-context-help=Access the data contained in the AssetTagSearchFacetDisplayContext object (for example, its isNothingSelected returns true if terms have not been selected).
```
Comment (jonmak08) [inline modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: Grammar seems weird?  

Should it read:
(for example, ~~its~~ isNothingSelected returns true if terms have not been selected).

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties (hunk 2 above)
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: You're right! I'll update this property.

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties (hunk 3 above)
Comment (jonmak08) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: I think there was another example of that weird grammar in another pull @thektan sent too.  Can you double check the other "help" text to make sure they sound ok?  Thanks!

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties (hunk 1 above)
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: It looks like on portal, this description is more abbreviated? I can update it to match.

image

File: modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties (hunk 3 above)
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties]: Will do, thanks Jon!

**Comment by oliv-yu** [comment]
ci:test:sf:norebase

**Comment by oliv-yu** [comment]
ci:test:search:norebase

**Comment by thektan** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84724. [See changes here.](https://github.com/thektan/liferay-portal/compare/4e117eeaee8c~2...4e117eeaee8c)

**Comment by thektan** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84725. [See changes here.](https://github.com/thektan/liferay-portal/compare/f9f158466887~3...f9f158466887)

**Comment by thektan** [comment]
Pull request submitted to https://github.com/thektan/liferay-portal/pull/147. [See changes here.](https://github.com/thektan/liferay-portal/compare/f9f158466887~3...f9f158466887)

**Pull request closed without merging**